assets/src/edit-story/karma/fixture/containers/designPanel/layers.js

```diff
@@ -39,4 +39,8 @@ export class Layers extends AbstractPanel {
   get layers() {
     return getAllByTestId(this.layersList, 'layer-option');
   }
+
+  get resizeHandle() {
+    return getByLabelText(this.node.ownerDocument, /Set panel height/i);
```

Comment:
Wouldn't all resizable panels have a handle? We don't have a lot, but this could fail. Maybe we could just go up a parent or two to find it?

Reply:
Hm, I removed the `ownerDocument` part and left just `this.node`- it looks just from this specific panel and should work as expected like this.